bin/elixir.bat

```diff
@@ -99,14 +99,39 @@ for  /d %%d in ("%originPath%..\lib\*.") do (
   set ext_libs=!ext_libs! -pa "%%~fd\ebin"
 )
 SETLOCAL disabledelayedexpansion
+
+rem ******* detect ANSI terminal support ********************
+set ERL=
+timeout 0 2>nul >nul || goto :run
```

Comment:
`timeout` here is used to detect if the script is running in a terminal or not, similar to `test -t 1 -a -t 2;` in bash. See http://stackoverflow.com/questions/22726198/how-to-test-if-stdin-is-terminal-in-batch.